src/handlers/cron_handlers/create.js

```
@@ -0,0 +1,224 @@
+  statusIndicator.start('Generating extend command for cron job')
+  const extendCmd = generateExtendCmd(extendOptions)
+  logger.debug(`Successfully generated cron job extend command: '${extendCmd}'`)
+  statusIndicator.succeed()
+  const hostList = codiusStateJson.hostList
+
+  if (!options.assumeYes) {
+    console.info(`Creating cron job to extend manifest every ${extendOptions.duration} seconds with the following command: ${extendCmd}`)
+    console.info('on the following host(s):')
+    jsome(hostList)
+    console.info('with the current status:')
+    jsome(statusDetails)
+
+    const userResp = await inquirer.prompt([
+      {
+        type: 'confirm',
+        name: 'continueToCronExtend',
+        message: `Do you want to continue creating the cron job for extending the pod?`,
+        default: false
+      }
+    ])
+    if (!userResp.continueToCronExtend) {
+      statusIndicator.start('User declined to create cron job')
+      throw new Error('Cron Extend aborted by user')
+    }
+  }
+
+  const cronJob = await addCronJob(extendCmd, codiusStateJson.manifestHash)
+  console.info('Successfully created new extend cron job:')
+  jsome(cronJob)
+  console.info('Saving cron job to Codius State File')
+  const savedCronJobs = codiusStateJson.status.cronJobs
+  if (savedCronJobs && savedCronJobs.length) {
+    savedCronJobs.push(cronJob)
+  } else {
+    codiusStateJson.status.cronJobs = [cronJob]
+  }
+  await fse.writeJson(codiusStateFilePath, codiusStateJson)
+  statusIndicator.succeed(`Codius State File: ${codiusStateFilePath} updated`)
+}
+
+async function checkCodiusInstallation () {
+  try {
+    statusIndicator.start('Testing global installation of codius cli')
+    await spawn('codius', ['--version'])
+    statusIndicator.succeed()
+  } catch (err) {
+    statusIndicator.fail()
+    logger.error(`Unable to run codius cli commands. Check global installation.`)
+    logger.error(`Test command: codius --version`)
```

You can just throw an error here instead of doing the .fail with the full message in the logs above then it will be caught in the createCron try catch rather than duplicating that logic here.